src/main/java/tracko/logic/commands/order/EditOrderCommand.java

```diff
@@ -261,6 +264,10 @@ public void updateItemList(Order orderToEdit, ItemQuantityPair itemToEdit) throw
                     orderedItems.set(i, updatedItem);
                     hasItemBeenUpdated = true;
                     break;
+                } else {
+                    // else case where edited quantity = original quantity, throw new exception to warn of
+                    // no quantity change
+                    throw new CommandException(MESSAGE_NO_CHANGE_IN_QUANTITY);
```

Comment:
I think I mentioned something about this in the DG for `edito` implementation, I think I mentioned that nothing will happen if the quantity is the same; do you mind helping me change it?

Reply:
Ah actually for this, should we allow the edit to go through if the quantity is the same? 
I was thinking about just setting
```java
hasItemBeenUpdated = true;
```
when quantity is the same. In this case the user will still see an item edited successfully confirmation message even when there are potentially no items actually edited.

Reply:
I think adding an exception is fine actually, so the user knows that the value is actually the same; otherwise the user won't know that the quantity has actually alr been updated and flag it as a bug (bc it doesnt change!)

Reply:
ok, thank you :) 

I've updated the edito section in the DG to say this: 
```java
Otherwise, a `CommandException` with `MESSAGE_NO_CHANGE_IN_QUANTITY` will be thrown. This is to warn users that the newly inputted `Quantity` is the same as the existing `Quantity` and does not update the `Quantity` field at all. 
```